app/src/main/java/com/expidev/gcmapp/http/GmaApiClient.java

```diff
@@ -357,7 +364,7 @@ public JSONObject getDetailsForMeasurement(@NonNull final String measurementId,
         // build request
         final Request<Session> request = new Request<>(MEASUREMENTS + "/" + measurementId);
         request.params.add(param("ministry_id", ministryId));
-        request.params.add(param("mcc", mcc));
+        request.params.add(param("mcc", mcc.toLowerCase()));
```

Comment:
good to know, in my assignments branch I've started switching to an enum to represent MCC's, but hadn't done much testing against the API yet to figure this out.